### PR TITLE
[CI] trigger library tests on cc changes

### PR DESCRIPTION
**Pull request by bracevac**
Changes to the capture checker should automatically trigger the scala2 library test.

I'm using this draft PR to test the triggers.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -179,6 +179,13 @@ jobs:
            github.event_name == 'pull_request'
            && contains(github.event.pull_request.body, '[test_scala2_library_tasty]')
          )
+         || (
+           github.event_name == 'pull_request'
+           && (   contains(github.event.pull_request.changed_files, 'scala2-library-cc/') 
+               || contains(github.event.pull_request.changed_files, 'scala2-library-cc-tasty/')
+               || contains(github.event.pull_request.changed_files, 'compiler/src/dotty/tools/dotc/cc/')
+              )
```
[the remark this answers is not in the document]
Comment (bracevac) [reply .github/workflows/ci.yaml:187]: I was thinking about this, but why would that be necessary? I don't see how changing the capture tests could affect the outcome of the standard library tests.

[the remark this answers is not in the document]
Comment (noti0na1) [reply .github/workflows/ci.yaml:187]: To make sure the modified/added test has the same behaviour/output with the two versions of stdlib.

[the remark this answers is not in the document]
Comment (bracevac) [reply .github/workflows/ci.yaml:187]: PS: Speaking of capture tests: they are spread out throughout the tests folder, according to `grep -nHr captureChecking ./tests`. Should we put them under a central folder? @odersky

[the remark this answers is not in the document]
Comment (bracevac) [reply .github/workflows/ci.yaml:187]: PPS: Those specific lines in the yaml file do not appear to work as intended. ChatGPT lied to me 😢

**Pull request closed without merging**